### ISSUE TITLE
Key Maps are wrong

### ISSUE THREAD

**Issue by angelitaooo** [Microsoft/vscode#16356]
- VSCode Version: 1.7.2 (1.7.2)
- OS Version: 10.12.1

Steps to Reproduce:

1. Type `CMD + -`
I expect the editor to zoom out but it comments out my code. The editor is mapping the keys in a wrong way.

![captura de pantalla 2016-12-01 a las 9 26 23 p m](https://cloud.githubusercontent.com/assets/18565471/20821120/f46b5268-b80d-11e6-982b-fadae06766c8.png)

**Comment by angelitaooo** [comment]
Basically, VS Code thinks that my keyboard is an english keyboard for key bindings `cmd + anything` but works as a regular spanish keyboard for writing code.

**Comment by chrmarti** [comment]
Related: #16157

**Comment by alexandrudima** [comment]
Starting with today's insiders version (i.e. after PR #22894 aka #17521) we have changed how keybinding dispatching works on OSX from using `e.keyCode` to using `e.code` (i.e. scan codes). 

I have installed the Spanish - ISO keyboard layout i.e.
![screen shot 2017-03-24 at 12 41 03](https://cloud.githubusercontent.com/assets/5047891/24292817/2d3e152e-108f-11e7-8383-91cd0822e366.png)

And I have verified that pressing:
* `cmd+shift+7` toggles line comments
* `cmd++` zooms in
* `cmd+-` zooms out
using the default keyboard shortcuts we ship with.
